pkg/chunk/encoding/bigchunk_test.go

```diff
@@ -12,12 +12,11 @@ import (
 func TestSliceBiggerChunk(t *testing.T) {
 	var c Chunk = newBigchunk()
 	for i := 0; i < 12*3600/15; i++ {
-		cs, err := c.Add(model.SamplePair{
+		_, err := c.Add(model.SamplePair{
```

Comment:
Lets check this field is nil (ie add a `require.Nil(t, nc)`)

Reply:
Ditto in all the other places you've used `_`.

Reply:
Done